docs/source/blog/arc-collaboration.md

```diff
@@ -0,0 +1,90 @@
+These plots can be added to existing figure axes you have created, and you can pass them the same formatting arguments as you would to the appropriate `matplotlib.pyplot`.
+Currently, the submodule has two wrappers to use;
+
+- `plot_centroid_trajectory`, which creates a plot of the trajectory of a given keypoint (or the centroid of a collection of keypoints) with a single line of code. Introduced in [#394](https://github.com/neuroinformatics-unit/movement/pull/394).
+- `plot_occupancy`, which creates an occupancy plot of an individual, given its position data. Collections of individuals are aggregated over, and if multiple keypoints are provided, the occupancy of the centroid is calculated. Introduced in [#403](https://github.com/neuroinformatics-unit/movement/pull/403).
+
+Examples to showcase the use of these plotting functions are currently [being produced](https://github.com/neuroinformatics-unit/movement/issues/415).
+[Our other examples](https://movement.neuroinformatics.dev/examples/index.html) have also been updated to use these functions where possible.
+
+### Broadcasting Methods
+
+Most of the time during analysis of experimental data, we want to repeat the same action multiple times across different coordinates, or individuals, or time-points.
+The data-structures that `movement` relies on - and the analysis methods it provides - go a long way to making this kind of analysis easy, thanks to array broadcasting.
+However when it comes to writing your own, custom analysis functions, most of the time it is much easier to write a function that takes in a single input, rather than writing an appropriate function that acts on an entire `DataArray`.
+But this approach typically means that you're then stuck writing a `for` loop over the values in your `DataArray`, and on top of that having to copy across the dimension labels or coordinates into the new array you're making as the `for` loop runs.
+
+With this in mind, we have decided to expose the {func}`movement.utils.broadcasting.make_broadcastable` decorator, which can turn functions that take in one-dimensional data (e.g., a single spatial coordinate, or a single time-point) into functions that act across `DataArray`s with appropriate dimensions.
+You can find a new {ref}`example <sphx_glr_examples_broadcasting_your_own_methods.py>` on how to use this decorator on your own functions.
+
+### Regions of Interest (RoIs)
+
+One of the biggest feature introductions to `movement` during this period is {mod}`support for RoIs <movement.roi>`.
+We now support one-dimensional regions of interest (segments, or piecewise-linear structures formed of multiple segments) as well as two-dimensional polygonal regions.
+You can create RoIs using the {class}`movement.roi.LineOfInterest` and {class}`movement.roi.PolygonOfInterest` classes respectively.
```

Comment:
These class links don't work properly. I'd do something lile this:

```md
{class}`PolygonOfInterest <movement.roi.polygon.PolygonOfInterest>`
```

Reply:
Weird that `sphinx` can't seem to follow imports from `__init__`, but I guess there's good reason for that!